src/components/algolia-search/index.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react'
+import {
+  AutocompleteOptions,
+  AutocompleteState,
+  createAutocomplete,
+} from '@algolia/autocomplete-core'
+import { Hit } from '@algolia/client-search'
+import { IconSearch16 } from '@hashicorp/flight-icons/svg-react/search-16'
+import { IconX16 } from '@hashicorp/flight-icons/svg-react/x-16'
+import { IconSlashSquare16 } from '@hashicorp/flight-icons/svg-react/slash-square-16'
+import { IconCornerDownLeft16 } from '@hashicorp/flight-icons/svg-react/corner-down-left-16'
+import useFocusOnKeyClick from 'hooks/use-focus-on-key-click'
+import SearchResultsLegend from './components/search-results-legend'
+import HitWrapper from './components/hit-wrapper'
+import { useAlgoliaNavigatorNext } from './lib/use-algolia-navigator-next'
+
+import s from './algolia-search.module.css'
+
+export type AutocompleteProps<THit extends Hit<unknown>> = Partial<
```

Comment:
I think there is a duplicate definition of this in the `algolia-search/types` file.